rules-reviewed/jboss-eap/weblogic/weblogic.windup.xml

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0"?>
-<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic_rules_general">
-   <metadata>
-      <description>
-         This ruleset provides analysis of WebLogic proprietary classes and constructs that may require individual attention when migrating to JBoss EAP 6+.
-      </description>
-      <dependencies>
-         <addon id="org.jboss.windup.rules,windup-rules-javaee,2.0.1.Final"/>			
-         <addon id="org.jboss.windup.rules,windup-rules-java,2.0.0.Final"/>
-      </dependencies>
-      <sourceTechnology id="weblogic" versionRange="(10,12]"/>
-      <targetTechnology id="eap" versionRange="[6,)"/>
-      <tags>
-         <tag>reviewed-2015-05-21</tag>
-      </tags>
-   </metadata>
-   <rules>
-   </rules>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic_rules_general"
```

Comment:
I don't think this file should be included. It will conflict with the actual work on weblogic.windup.xml